### DISCUSSION TITLE
Step-by-step guide to contributing on GitHub

### DISCUSSION THREAD

**Discussion by WebShivam** [EddieHubCommunity/support#1456] [First-Timers] [answered]
This is the article I read today and I found it useful so I'm sharing with you all folks so you also can get benifitted by this

**Comment by WebShivam** [comment, marked as answer]
Here's the link
https://www.dataschool.io/how-to-contribute-on-github/

**Reply by Kernix13** [reply]
That's a good article - I read that a few weeks ago.

**Reply by segunajibola** [reply]
helped me, thanks

**Comment by Kernix13** [comment]
I also am building a repo on [git commands and contributing](https://github.com/Kernix13/beginner-git-commands). There was a freeCodeCamp article that helped me a lot: [how-to-setup-freecodecamp-locally.md](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/docs/how-to-setup-freecodecamp-locally.md).
